src/plugins/saved_objects_management/public/management_section/objects_table/components/show_duplicate_modal.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Any modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { I18nStart } from 'opensearch-dashboards/public';
+import { SavedObjectsDuplicateModal, ShowDuplicateModalProps } from './duplicate_modal';
+
+/**
+ * Represents the result of trying to duplicate the saved object.
+ * Contains `error` prop if something unexpected happened (e.g. network error).
+ * Contains an `id` if persisting was successful. If `id` and
+ * `error` are undefined, persisting was not successful, but the
+ * modal can still recover (e.g. the name of the saved object was already taken).
+ */
+
+export function showDuplicateModal(
```

Comment:
This file is used for?

Reply:
This file is used to reuse the duplicate logic in subsequent dashboard and visualize plugins.

Reply:
If we need to show a modal, we can embed it as a React component. Why do we make it as a util?

Reply:
The file has been deleted.